sources/rest_api/__init__.py

```diff
@@ -211,6 +211,9 @@ def create_resources(
         request_params = endpoint_config.get("params", {})
         request_json = endpoint_config.get("json", None)
         paginator = create_paginator(endpoint_config.get("paginator"))
+        row_filter = endpoint_resource.pop("filter", None)
```

Comment:
Add an identity function that just returns the input or is an effective no op for each functional param. This removes the need for the ternaries in the function body.

Reply:
done